bolt.rb

```diff
@@ -39,15 +69,88 @@ def list
   end
 end
 
+
 class Target < Wash::Entry
   label 'target'
-  state :connection_info
+  parent_of VOLUMEFS
+  description <<~DESC
+    This is a target. You can view target configuration with 'metadata', and SSH
```

Comment:
Ah I think this is where I meant to say "Did you mean the `meta` command"

Reply:
Ah yes. I'll update it.